.github/workflows/release.yml

```diff
@@ -565,25 +584,28 @@ jobs:
 
         echo "::group::Build"
         echo "Running on $(sysctl -n hw.logicalcpu) cores"
-        make -j$(sysctl -n hw.logicalcpu)
+        cmake --build . -j $(sysctl -n hw.logicalcpu)
         echo "::endgroup::"
 
     - name: Build package
       run: |
         cd build-x64
 
+        echo "::group::Create universal binary"
```

Comment:
The other steps you called `Create bundles`, but this is called `Build package`. Maybe name them all the same? (I don't care which, btw, just having more things the same helps future-us I think :D)